modules/services/workspace-service/src/test/java/org/ballerinalang/composer/service/workspace/langserver/compilationunit/TopLevelCompletions.java

```diff
@@ -0,0 +1,37 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+package org.ballerinalang.composer.service.workspace.langserver.compilationunit;
+
+import org.ballerinalang.composer.service.workspace.langserver.CompletionTest;
+import org.testng.annotations.DataProvider;
+
+import java.io.File;
+
+/**
+ * Top level completion tests (compilation unit level)
+ */
+public class TopLevelCompletions extends CompletionTest {
+    @DataProvider(name = "completion-data-provider")
+    @Override
+    public Object[][] dataProvider() {
+        return new Object[][] {
+//                {"topLevelEmptyFirstLine.json", "completions" + File.separator + "function" + File.separator},
```

Comment:
Shall we remove this commented line ?

Reply:
This test will be enabled when the particular fix is added with a separate commit